package/yast2-theme.spec

```diff
@@ -121,6 +121,7 @@ mv %{buildroot}%{yast_themedir}/SLE %{buildroot}%{yast_themedir}/current
 cp theme/SLE/wmconfig/* %{buildroot}/etc/icewm/
 # SLE doesn't have oxygen5-icon-theme
 rm -rf %{buildroot}%{yast_icondir}/oxygen %{buildroot}%{yast_icondir}/breeze
+rm -rf %{buildroot}%{yast_icondir}/oxygen %{buildroot}%{yast_icondir}/breeze-dark
```

Comment:
Both breeze and breeze-dark need to be removed

Reply:
argh...
I thought it was one line per theme...